### DISCUSSION TITLE
how to save g711 and h264 to mpegts file?

### DISCUSSION THREAD

**Discussion by Mosphere** [bluenviron/gortsplib#504] [Questions]

**Comment by aler9** [comment]
Hello, it's not possible to save G711 tracks to MPEG-TS files since currently there isn't any specification that tells how to mux G711 tracks into MPEG-TS, and even if it was, there isn't any player that supports playing G711 tracks from MPEG-TS.

You have to re-encode the track with another codec (AAC?) by using FFmpeg or GStreamer.
